hm_pyhelper/miner_param.py

```diff
@@ -101,7 +101,7 @@ def get_gateway_mfr_version() -> Version:
         raise GatewayMFRInvalidVersion(err_str).with_traceback(e.__traceback__)
 
 
-def get_gateway_mfr_command(sub_command: str) -> list:
+def get_gateway_mfr_command(sub_command: str, slot: int = 0) -> list:
```

Comment:
it is alright for the time being, no need to change. But slot kind of goes against SWARM_KEY_URI as it always overrides the slot it provides.

Reply:
Yes, I didn't want to change hardware definitions as they are used in other parts of code too. Ideally slot should not have been part of SWARM_KEY_URI.